src/overlays/actors/ovl_Bg_Dodoago/z_bg_dodoago.c

```diff
@@ -17,10 +17,10 @@ void BgDodoago_Destroy(Actor* thisx, GlobalContext* globalCtx);
 void BgDodoago_Update(Actor* thisx, GlobalContext* globalCtx);
 void BgDodoago_Draw(Actor* thisx, GlobalContext* globalCtx);
 
-void func_80871CF4(BgDodoago* this, GlobalContext* globalCtx);
-void func_80871FB8(BgDodoago* this, GlobalContext* globalCtx);
+void BgDodoago_WaitExplosives_(BgDodoago* this, GlobalContext* globalCtx);
```

Comment:
accidental trailing underscore

Reply:
that's how i indicate a not-final name when I work on docs, but yeah accidentally left it in :+1: